app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -138,6 +166,43 @@ export class PublishRepository extends React.Component<
     )
   }
 
+  private renderTeams(): JSX.Element | null {
+    if (this.state.teams.length === 0) {
+      return null
+    }
+
+    const options = new Array<JSX.Element>()
+    options.push(
+      <option value={-1} key={-1}>
+        None
```

Comment:
What happens if there’s a team called “None?”

Reply:
It's a potential source of confusion, but basically:

 - It'll appear below this in the list
 - It'll have it's own distinct `key` derived from it's index, which we should correctly map when the select it

I'm open to suggestions about what to put here for the "i don't want to select a team" thing here, but I've borrowed this form the org selection list.

Somewhat related: GitHub The Website does it a bit differently when creating a new repository, showing your user account as the owner by default when you're creating a new repository. 

<img width="549" src="https://user-images.githubusercontent.com/359239/38069260-221226ae-3361-11e8-8280-489c5cac01fb.png">

Maybe we could make this experience be more consistent with that?